lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -16,8 +15,8 @@ class Standardizer(ABC):
     transforming specific fields within a document according to certain rules.
     """
 
-    @abstractmethod
-    def fixer(self, text: str) -> Union[str, Tuple[str, date]]:
+    @staticmethod
```

Comment:
Does this mean that all standardizer implementations have to be static? I guess that's true now, but not sure it's always true. That said, I guess this abstraction doesn't provide us all that much anyway, so whatever.

Reply:
Yes, for now. I could not figure out how to get mypy to be happy with the method being non-static in the base class but static in the subclass.